feat: add watermark to PDF command

MohamedK1 opened the pull request:
## Description
Extending the extension with add watermark command
where you specify your watermark text and configure the following options:
- pages to be watermarked
- position of watermark or Mosaic to do it in 9 combinations of position.
- The layer of the watermark (above or below the PDF)
- angle of rotation
- transparency
- font/font size/font style

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

https://github.com/raycast/extensions/assets/41074872/9d9064b9-b4a7-446c-8ccf-87a34981a4a2


<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

mil3na commented:
Hello, hello!

This command did not work for me. I got the same, not very useful, 400 error:
![CleanShot 2024-07-04 at 14 35 27](https://github.com/raycast/extensions/assets/611002/aca19276-4f7c-4218-b45e-75c5a3bebd86)

Also, here it says Transparency, but the tooltip talks about opacity. Which one is it?
![CleanShot 2024-07-04 at 14 34 49](https://github.com/raycast/extensions/assets/611002/019d3db8-ebeb-4909-81a9-ce72339af2fe)

MohamedK1 commented:
Hey @mil3na 👋 ,

> This command did not work for me. I got the same, not very useful, 400 error:

Yep, there was an issue for Regular text. It's due to unfortunately outdated documentation. fixed it and will opened an issue in ILovePDF side.

For the error message, I updated this PR too with the same util used for unlock. Will use it in all other commands after landing.

I also updated the transparency to be Opacity. Sorry for confusion.

mil3na reviewed (approved):
LGTM
